app.json

```diff
@@ -11,9 +11,15 @@
       "resizeMode": "contain",
       "backgroundColor": "#ffffff"
     },
-    "assetBundlePatterns": [
-      "assets/*"
+    "plugins": [
+      [
+        "expo-image-picker",
+        {
+          "photosPermission": "The app accesses your photos to let you share them with your friends."
```

Comment:
this message should be changed.

Reply:
Done

Reply:
You can change the message like: Grant Qur8 permission to access your photo gallery for a richer experience!